Extension/src/LanguageServer/Providers/onTypeFormattingEditProvider.ts

```diff
@@ -17,7 +17,7 @@ export class OnTypeFormattingEditProvider implements vscode.OnTypeFormattingEdit
         return new Promise<vscode.TextEdit[]>((resolve, reject) => {
```

Comment:
Same as previous comment.  I think this provider API override can be made async.